[2021-10-15 18:34:32] [WARNING]: lambda 初始化失败: Post "http:///runtime/init/ready": http: no Host in request URL 
[2021-10-15 18:34:32] [WARNING]: 警告: pprof 服务不支持鉴权, 请不要运行在公网. 
[2021-10-15 18:34:34] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 18:36:01] [WARNING]: 监听反向WS API时出现错误: websocket: close 1006 (abnormal closure): unexpected EOF 
[2021-10-15 18:36:07] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:65215->127.0.0.1:8080: use of closed network connection 
[2021-10-15 18:36:12] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 18:36:17] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 18:37:14] [WARNING]: 监听反向WS API时出现错误: websocket: close 1006 (abnormal closure): unexpected EOF 
[2021-10-15 18:37:17] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:61566->127.0.0.1:8080: use of closed network connection 
[2021-10-15 18:37:22] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 18:41:00] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:50725->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-15 18:41:02] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:50725->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-15 18:41:07] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 18:48:46] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 18:48:51] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 19:00:30] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 19:00:35] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 19:14:11] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 19:14:16] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 19:21:17] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 19:21:22] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 19:24:34] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:58829->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-15 19:24:37] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:58829->127.0.0.1:8080: use of closed network connection 
[2021-10-15 19:24:42] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 19:24:47] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 19:24:52] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 19:24:57] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 19:25:02] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 19:25:07] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 19:25:12] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 19:31:27] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 19:31:32] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 19:36:48] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 19:36:54] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 19:55:54] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 19:55:59] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 20:00:03] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 20:00:08] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 20:24:25] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 20:24:31] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 20:54:33] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:54727->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-15 20:54:37] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:54727->127.0.0.1:8080: use of closed network connection 
[2021-10-15 20:54:42] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:55:01] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:54250->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-15 20:55:02] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:54250->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-15 20:55:07] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:55:12] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:55:17] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:55:22] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:55:27] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:55:41] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:54261->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-15 20:55:42] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:54261->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-15 20:55:47] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:55:52] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:55:57] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 20:56:02] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 21:18:26] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 21:18:31] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 21:19:29] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:54275->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-15 21:19:32] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:54275->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-15 21:19:37] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 21:19:42] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 21:21:34] [WARNING]: 获取信息时出现错误: leveldb: not found id: -203902517 
[2021-10-15 21:42:30] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:57490->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-15 21:42:32] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:57490->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-15 21:42:37] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 21:42:42] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-15 22:22:07] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 22:22:12] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 22:50:33] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 22:50:38] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 23:21:32] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 23:21:38] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 23:35:47] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 23:35:52] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 23:36:49] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 23:36:55] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 23:52:52] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 23:52:58] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-15 23:53:15] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-15 23:53:20] [WARNING]: 群消息发送失败: 账号可能被风控. 
